### PR TITLE
Changed mapping tester to a data service

### PR DESCRIPTION
**Pull request by rjrudin**
This will cause a merge conflict with the develop branch in build.gradle, but it'll be simple to clean up. If you rebase your nested-entities branch off develop, I can resubmit this.

### CODE REVIEW

File: marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/MappingService.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+              return BaseProxy.JsonDocumentType.toJsonNode(
+                baseProxy
+                .request("testMapping.sjs", BaseProxy.ParameterValuesKind.MULTIPLE_ATOMICS)
+                .withSession()
+                .withParams(
+                    BaseProxy.atomicParam("docURI", false, BaseProxy.StringType.fromString(docURI)),
+                    BaseProxy.atomicParam("mappingName", false, BaseProxy.StringType.fromString(mappingName)),
+                    BaseProxy.atomicParam("mappingVersion", false, BaseProxy.StringType.fromString(mappingVersion)))
+                .withMethod("POST")
+                .responseSingle(false, Format.JSON)
```
Comment (srinathgit) [inline marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/MappingService.java:45]: This is a GET method. Should the .api file be changed and the this file be regenerated ?

**Review by srinathgit — CHANGES_REQUESTED** [review]

File: marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/MappingService.java (hunk 1 above)
Comment (rjrudin) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/MappingService.java:45]: I believe Data Services always generates  POST, I don't think there's a way to modify that - at least I don't see it in the docs. But does it matter? The client doesn't need to know.

Comment (srinathgit) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/MappingService.java:45]: It doesn't matter, I will change the code to use POST method

Comment (rjrudin) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/MappingService.java:45]: I'm not sure what you mean - the client to the interface doesn't care what HTTP method is used, and the Data Services endpoint doesn't care either. The Spring Boot controller can still be a "GET" request.

Comment (srinathgit) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/MappingService.java:45]: Ok, I thought I had to change the Spring Boot controller. It works fine without any changes.

**Pull request merged**